hair.css

```diff
@@ -0,0 +1,251 @@
+header {
+  background-color: coral;
+  width: 100%;
+  height: 15%;
+  display: flex;
+  /* flex-flow: row nowrap; */
+  justify-content: center;
+  /* align-items: center; */
+  margin-bottom: 80px;
+}
+
+.headerbox {
+  width: 50%;
```

Comment:
i think this makes it too small. it would be better if you set a max-width. try
`width: 100%;`
`max-width: 1080px;`
or `max-width: 1444px;`